modules/data-mover-for-csi-snapshots.adoc

```diff
@@ -0,0 +1,216 @@
++
+The OADP operator installs two custom resource definitions (CRDs), `VolumeSnapshotBackup` and `VolumeSnapshotRestore`.
++
+.Example `VolumeSnapshotBackup` CRD+
+[source,yaml]
+----
+apiVersion: datamover.oadp.openshift.io/v1alpha1
+kind: VolumeSnapshotBackup
+metadata:
+  name: <vsb_name>
+  namespace: <namespace_name> <1>
+spec:
+  volumeSnapshotContent:
+    name: <snapcontent_name>
+  protectedNamespace: <adp_namespace>
+  resticSecretRef:
+    name: <restic_secret_name>
+----
+<1> Specify the namespace where the volume snapshot resides.
++
+.Example `VolumeSnapshotRestore` CRD+
+[source,yaml]
+----
+apiVersion: datamover.oadp.openshift.io/v1alpha1
+kind: VolumeSnapshotRestore
+metadata:
+  name: <vsr_name>
+  namespace: <namespace_name> <1>
+spec:
+  protectedNamespace: <protected-ns> <2>
+  resticSecretRef:
+    name: <restic_secret_name>
+  volumeSnapshotMoverBackupRef:
+    sourcePVCData:
+      name: <source_pvc_name>
+      size: <source_pvc_size>
+    resticrepository: <your_restic_repo>
+    volumeSnapshotClassName: <vsclass_name>
+----
+<1> Specify the namespace where the volume snapshot exists.
+<2> Specify the namespace where the Operator is installed. The default is `openshift-adp`.
+
+. You can back up a volume snapshot by performing the following steps:
+
+.. Create a backup CR:
++
+[source,yaml]
+----
+apiVersion: {velero-domain}/v1
+kind: Backup
+metadata:
+  name: <backup_name>
+  namespace: <protected_ns> <1>
+spec:
+  includedNamespaces:
+  - <app_ns>
+  storageLocation: velero-sample-1
+----
+<1> Specify the namespace where the Operator is installed. The default namespace is `openshift-adp`.
+
+.. Wait up to 10 minutes and check whether the `VolumeSnapshotBackup` CR status is `Completed` by entering the following commands:
+
++
+[source,terminal]
+----
+$`oc get vsb -n <app_ns>`
+----
++
+$`oc get vsb <vsb_name> -n <app_ns> -ojsonpath="{.status.phase}`
+----
+
++
+
++
+A snapshot is created in the object store that was specified in the Restic secret.
++
+[NOTE]
+====
+If the status of the `VolumeSnapshotBackup` CR becomes `Failed`, refer to the Velero logs for troubleshooting.
+====
+
+. You can restore a volume snapshot by performing the following steps:
+
+.. Delete the application namespace and the `volumeSnapshotContent` that was created by the Velero CSI plug-in.
+
+.. Create a `Restore` CR and set `restorePVs` to `true`.
++
+.Example `Restore` CR
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: <restore-name>
+  namespace: <protected-ns>
+spec:
+  backupName: <previous-backup-name>
+  restorePVs: true
+----
+
+.. Wait up to 10 minutes and check whether the `VolumeSnapshotRestore` CR status is `Completed` by entering the following command:
++
+[source,terminal]
+----
+$`oc get vsr -n <app-ns>`
+----
++
+[source,terminal]
+$`oc get vsr <vsr-name> -n <app-ns> -ojsonpath="{.status.phase}`
+----
```

Comment:
```suggestion
[source,terminal]
----
$ oc get vsr -n <app-ns>
----
+
[source,terminal]
----
$ oc get vsr <vsr-name> -n <app-ns> -ojsonpath="{.status.phase} 
----
```
Again, I think this needs a close quote on `"{.status.phase}`. No backticks on codeblocks and you need a space between `$ oc get`

Reply:
See oc CLI reference: https://docs.openshift.com/container-platform/4.11/cli_reference/openshift_cli/developer-cli-commands.html#oc-config-view

`-o jsonpath`